lib/foreman_remote_execution_core/polling_script_runner.rb

```diff
@@ -0,0 +1,133 @@
+module ForemanRemoteExecutionCore
+  class PollingScriptRunner < ScriptRunner
+
+    def initialize(options = {})
+      super(options)
+      @callback_host = options[:callback_host]
+      @task_id = options[:uuid]
+      @step_id = options[:step_id]
+      try_obtain_otp!
+    end
+
+    def refresh_interval
+      60
```

Comment:
even if it was just a global setting for the plugin, that would be better than hard coded at 60sec